frontend/src/components/Map/index.js

```diff
@@ -1,4 +1,3 @@
-import chroma from "chroma-js";
```

Comment:
fwiw i'm pretty sure we can remove this from the whole project, but let's take care of one thing at a time :P